src/webhook.rs

```diff
@@ -32,17 +35,31 @@ impl Webhook {
         }
     }
 
-    pub async fn execute(
+    pub async fn alert(
         &self,
-        location: &Location,
-        content: impl Display,
+        errors: &[(&Location, HandleLocationError)],
     ) -> Result<(), WebhookExecuteError> {
+        let mut embed = EmbedBuilder::new()
+            .color(0x9E2C2C)
+            .description("Some errors occurred.\nWe notify again when everything is green again.");
+
+        for field in errors.iter().take(FIELD_COUNT).map(|(location, error)| {
+            EmbedFieldBuilder::new(location.name, error.to_string()).build()
+        }) {
+            embed = embed.field(field);
+        }
+
+        self.execute_embed_webhook(embed.build()).await
+    }
+
+    pub async fn resolved(&self) -> Result<(), WebhookExecuteError> {
         let embed = EmbedBuilder::new()
-            .color(0x9e2c2c)
-            .author(EmbedAuthorBuilder::new(location.name).build())
-            .description(content.to_string())
-            .build();
+            .color(0x57F287)
+            .description("Every request was successful.");
```

Comment:
This message only shows that every request was successful but does not explain that the collector works as expected again
```suggestion
            .description("All requests have been successful. Collector working as expected again.");
```